code/datums/uplink_item.dm

```diff
@@ -362,6 +367,9 @@ var/list/uplink_items = list()
 	item = /obj/item/weapon/card/emag
 	cost = 6
 
+/datum/uplink_item/device_tools/emag/new_uplink_item(new_item, turf/location, mob/user)
+	new new_item(location, 1) //Uplink emags are infinite
```

Comment:
undocumented change?

Reply:
It's the fourth on the list of "changed bundle and uplink", emags used to be unintentionally infinite but they are now intentionally infinite